.stylelintrc

```diff
@@ -1,3 +1,20 @@
 {
-  "extends": "stylelint-config-standard"
+  "extends": "stylelint-config-standard",
+  "rules": {
+    "at-rule-empty-line-before": [
```

Comment:
Her er forøvrig stylelint-config-standard som vi extender:
https://github.com/stylelint/stylelint-config-standard/blob/master/index.js